scrapers/Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR /app
 
 COPY pyproject.toml pyproject.toml
 COPY poetry.lock poetry.lock
-RUN poetry env use python
+RUN poetry env use python3.10
```

Comment:
@linusha Any idea as to why this is required? Shouldn’t `python` link to `python3.10` in the container, i.e. `python -v` == `python3.10 -v`?

Reply:
Not really unfortunately. As I wrote above, this was necessary in order to get poetry to stop using the older python3.9 venv inside of the container. But I am a) not sure why it is necessary, as I would have had the same expectation as you and b) I am also confused as to why there still were older poetry venvs and stuff after deleting my scrapers container and rebuilding it without any caches.

Reply:
Maybe because we persist the venvs in a volume. Still, while building the image, `poetry install` should have been run… Will try it out later and see if this is an issue for me as well!